docs/src/configuration/app/build.md

```diff
@@ -138,45 +138,19 @@ What is "safe" to run in a `post_deploy` hook vs. in a `deploy` hook will vary b
 
 The `post_deploy` hook logs to its own file in addition to the activity log, `/var/log/post-deploy.log`.
 
-## How do I compile Sass files as part of a build?
```

Comment:
This is a common task and request.  We can update it if needed, but shouldn't drop it.

Reply:
Mh, weird.
The reasons why I've removed this are:
- If I have a `package.json` file, I don't see why I would run `grunt` on its own during the build process and not use npm scripts. I'd rather avoid having my build steps scattered between `package.json` and `.platform.app.yaml`, especially here, since grunt is a required dependency in order to build (a part of) my app, but `npm install` wouldn't install it because it's been installed before? What's the benefit in doing that? My `package.json` file becomes dependant on `.platform.app.yaml`.
- Involving a specific task runner just to compile Sass files seems a bit overkill and adds unnecessary assumptions.

Also, [Ruby sass is deprecated](https://sass-lang.com/ruby-sass) and even though Grunt is still alive and used (kinda), modern applications mostly rely on Webpack / Rollup / Parcel.

This example feels a bit convoluted and outdated.

If it's a common question, and to stay in the idea of this example, maybe this would work:
```yaml
dependencies:
  nodejs:
    sass: "^3.4.21"

hooks:
  build: |
    npm install
    npm run build-css
```
?

Reply:
Wait, Sass was fully ported from Ruby to JS?  Why not to a proper compiled language?  I missed this memo...

This tutorial was written several years ago, so it's quite possible that the common practices have shifted.  Last I looked, Grunt and Gulp were the common JS task runners.  Does no one use those anymore in favor of `npm run`?  How about non-JS people who are going to be more than 6 months behind JS conventions? :smile: 

I'm fine with updating the tutorial if the conventions have shifted, but we should keep Sass compilation as an example.  (It was nice as a multi-lingual example, but if that's not the current state of the art, meh.)

Reply:
It's written in [Dart](https://dart.dev/), so you can either use the binary or the JS version.

`npm run X` will just run the npm script `X` declared in your `package.json` in `scripts`. You can do anything in there (it's just shell commands). So you can rely on a task runner / bundler or run a binary directly. Here, saying `npm run build-css` just assumes that we have a `build-css` npm script that does what it needs to do in order to build the css. Whether it uses a task runner or the sass binary directly is an implementation detail. (It's called "npm" scripts but Yarn would work just as well by doing `yarn build-css`.)

Reply:
Adapted in 3eff045